### PR TITLE
Update pv move state

### PR DESCRIPTION
**Pull request by tasleson**
Ensure that the dbus objects have a fairly accurate representation of the state when a move is in progress.

### CODE REVIEW

**Comment by tasleson** [comment]
@vpodzime Please review when you get a chance, thanks!

File: daemons/lvmdbusd/utils.py [hunk 1]
```diff
@@ -510,16 +510,19 @@ def add_no_notify(cmdline):
...
-	if '--config' in cmdline:
-		for i, arg in enumerate(cmdline):
-			if arg == '--config':
-				cmdline[i] += "global/notify_dbus=0"
-				break
-	else:
-		cmdline.extend(['--config', 'global/notify_dbus=0'])
+	# Only after we have seen an external event will be disable lvm from sending
+	# us one when we call lvm
+	if cfg.ee:
```
Comment (vpodzime) [inline daemons/lvmdbusd/utils.py]: ``cfg.ee`` is quite cryptic here. It's not being used very often so maybe a more descriptive longer name wouldn't hurt. ;)

**Review by vpodzime — APPROVED** [review]
Looks good to me otherwise. Thanks for looking at this so fast!

**Comment by tasleson** [comment]
@vpodzime Thanks for the review, `ee` renamed to `got_external_event`.

**Pull request closed without merging**